spec/datadog/profiling/agent_settings_resolver_spec.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'datadog/profiling/agent_settings_resolver'
+require 'datadog/core/configuration/settings'
```

Comment:
On the one hand, this file is pointless. On the other hand, it feels odd to add a class without a unit test file for it. Happy to kill it.

Reply:
🪓 it! Nothing to test, nothing to see here.